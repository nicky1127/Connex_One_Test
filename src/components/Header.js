import React from 'react';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(() => ({
  root: {
    height: '8%',
    width: '100%',
    padding: '10px',
    display: 'flex'
  },
  title: {
    flex: '1 1 auto'
  },
  info: {
    flexBasis: '20%'
  }
}));

const Header = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <h1 className={classes.title}>Connex One - Full Stack Technical Test</h1>
      <div className={classes.info}>
        <h3>Applicant : Nicky Lai</h3>
        <h3>Date: 30/07/2020</h3>
      </div>
    </div>
  );
};

export default Header;
