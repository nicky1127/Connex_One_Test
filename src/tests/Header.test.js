import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from '../components/Header';

afterEach(cleanup);

describe('Header', () => {
  it('Snapshot matchs', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
