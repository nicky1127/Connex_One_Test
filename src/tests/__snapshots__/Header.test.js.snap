// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot matchs 1`] = `
<div
  className="makeStyles-root-1"
>
  <h1
    className="makeStyles-title-2"
  >
    Connex One - Full Stack Technical Test
  </h1>
  <div
    className="makeStyles-info-3"
  >
    <h3>
      Applicant : Nicky Lai
    </h3>
    <h3>
      Date: 30/07/2020
    </h3>
  </div>
</div>
`;
